### PR TITLE
Initialized bookstore

### PR DESCRIPTION
**Pull request by Boku52500**
- Installed React.
- Installed React Router v6.
- Used routers for navigation.
- Used react components.
- Fixed linters.

PLEASE READ THIS PART CAREFULY.
On my end none of the linters listed in last review come up. Not when I do npm start, neither when I run npx eslint . and npx eslint . --fix, so I am not sure how to fix this. Even in pull request they pass so I have no idea what to fix. I can not get a list of what the errors are. Here is proof
https://gyazo.com/75e46c3f81ec6b942d7374628a1bd0b1
https://gyazo.com/811192d65c0f8193225921d080528017
https://gyazo.com/4c2659800f790320a9bc8c0d57d7f4fa

### CODE REVIEW

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 

Hi @Boku52500, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

[REQUIRED] Errors are displayed when I open your code in the browser
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const BookList = ({books}) => {
+  console.log(books)
+  return (
+    <ul>
+      {books.map((book) => (
+        <li key={book.id}>
+            {book.title} {book.author}
+        </li>
+      ))}
```
Comment (Kamba56) [inline src/components/BookList.jsx:11]: - [Optional] Please add a button to remove the book from the list. The button functionality shouldn't be implemented yet but the markup should be in your file.

File: src/components/Books.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+import AddBook from './AddBook';
+import BookList from './BookList';
+
+const Books = () => {
+  const [booksList, setBooksList] = useState([]);
```
Comment (Kamba56) [inline src/components/Books.jsx:7]: - [Optional] Consider creating a dummy list of books for the sake of this exercise to make sure the list of books displays properly on your screen.

**Review by Kamba56 — APPROVED** [review]
Hi @Boku52500, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.jsx (hunk 1 above)
Comment (Boku52500) [reply src/components/BookList.jsx:11]: @Kamba56 Thank you for suggestions and review. I will be applying both of them in my next project for this repository.

File: src/components/Books.jsx (hunk 2 above)
Comment (Boku52500) [reply src/components/Books.jsx:7]: @Kamba56 Thank you for suggestions and review. I will be applying both of them in my next project for this repository.

**Pull request merged**